announcements/2020-Week-25/azure.md

```diff
@@ -0,0 +1,27 @@
+# :zap: This Week in Releases (June 19, 2020):zap:
+
+## Azure v11.3.3
+
+**If you are upgrading from 11.3.1 or 11.3.2, upgrading to this release will not roll your nodes. It will only update the apps.**
```

Comment:
This sounds not 100% correct, it does not update apps does it? What does it do?

Reply:
It does, 3 apps: metrics-server, net-exporter and kube-state-metrics

Reply:
ok, right, didn't check the changelog right. I guess you wanted to only mention the important things, and it confused me cause here we say it updates apps, and then we do not say anything about apps updated. But might be ok. Your choice

Reply:
maybe could add detail like `adds support for having the control plane and the tenant clusters in a different subnet to some of our monitoring components`

Reply:
do you want to include this note in the announcement of releases? I feel it belongs only to the actual release notes here https://github.com/giantswarm/releases/tree/master/azure/v11.3.3#zap--giant-swarm-release-1133-for-azure-zap